openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -58,16 +58,12 @@ def process(self, instance):
         else:
             members = instance[:]
 
-        loaded_containers = None
-        if set(self.add_for_families).intersection(
-                set(instance.data.get("families")),
-                set(instance.data.get("family").lower())):
-            loaded_containers = self._add_loaded_containers(members)
-
         selection = members
-        if loaded_containers:
-            self.log.info(loaded_containers)
-            selection += loaded_containers
+        if set(self.add_for_families).intersection(
+                set(instance.data.get("families"))) or \
+                instance.data.get("family") in self.add_for_families:
```

Comment:
Are we sure `family` is just a fallback and that `families` always includes what is in `family`?

With cleaning up "technical debt" before OP v4 I'd say removing `family` altogether and solely keeping `families` would be high on the list. :)

Reply:
I agree. But it needs to be done a different PR, otherwise it's untrackable. That being said we'll then have to rely on the first item in `families` to determine the main family of the subset. Also not ideal

Reply:
>  That being said we'll then have to rely on the first item in families to determine the main family of the subset. Also not ideal

Agreed - it's less explicit. I think it'd be good to identify exactly why we need to be able to identify a "primary family" as well.

Worst case we could have a `primaryFamily` data key that MUST also be present in `families`. (e.g. have a global validator so we make sure it's always the case if it exists) So that wherever check for family related things we can always rely on solely `families` and we just use `primaryFamily` to know what the primary is. We could also use the old `family` key for that - but might be harder fix the behavior for and it's less explicit.

Reply:
> Are we sure family is just a fallback and that families always includes what is in family?

Nevermind - I read the logic wrong. It actually does check both.

Reply:
I think we need to handle families in more robust way in v4 anyway and that should be another discussion.